WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -174,6 +175,42 @@ public static async Task ShellExecAsync(string cmd, bool waitForExit = true)
 		}
 	}
 
+	/// <summary>
+	/// Executes a command with Bourne shell and returns Standard Output.
+	/// </summary>
+	public static async Task<List<string>> ShellExecAndGetResultAsync(string cmd)
```

Comment:
Looks like a duplicate.

Reply:
The two functions can call each other?

Reply:
Not full duplicate, the other function doesn't give you back the result of the executed command what we need. But they can't call eachother because they have different return value.
If `ShellExecAndGetResultAsync` calls `ShellExecAsync` inside, it can't get back the result. To change the return value, I would need to change it everywhere in the code, even at places where it's not needed.